composables/content-parse.ts

```diff
@@ -618,7 +618,7 @@ function createTransformNamedMentions(mentions: mastodon.v1.StatusMention[]) {
       const href = node.attributes.href
       const mention = href && mentions.find(m => m.url === href)
       if (mention) {
-        node.attributes.href = `/${currentServer.value}/@${mention.acct}`
+        node.attributes.href = `https:\/\/${currentServer.value}/@${mention.acct}`
```

Comment:
The `href` being relative to the deployment is intentional, as we want the user to stay in Elk instead of going to the original server. We need to find a way to fix this without modifying this link.

Reply:
The `AccountHoverWrapper` is inserted if the href matches the regexp `UserLinkRE` here:
[UserLinkRE](https://github.com/elk-zone/elk/blob/main/composables/content-render.ts#L88-L94)

Unfortunately `UserLinkRE` starts with http/s, so the relative links are failing, and therefore no user card on hover.

What would be in your view the best solution:
* Add a new regexp without http/s at the start and test against it and append the AccountHoverWrapper
* Change the `UserLinkRE`  regexp (but that is used at another place as well, so I'm not preferring it)
* something else?

Reply:
So I have reverted back, and just changed the regex to work with or without http at the beginning. Looks like working.